app/static/tests/unit/components/SessionInitialiseModal.test.ts

```diff
@@ -0,0 +1,39 @@
+import { shallowMount } from "@vue/test-utils";
+import SessionInitialiseModal from "../../../src/app/components/SessionInitialiseModal.vue";
+
+describe("SessionInitialiseModal", () => {
+    const getWrapper = (open = true) => {
+        return shallowMount(SessionInitialiseModal, { props: { open } });
+    };
+
+    it("renders as expected when open is true", () => {
```

Comment:
are these tests necessary with all the css checks? I believe the main purpose of component tests is to yes do a quick check things are right but mainly functionality. But i would love to discuss with you and the team more as its a different philosophy of thought.

Reply:
I think it's ok here as it is testing the component logic, which will render different css and classes depending on a prop value.